### ISSUE TITLE
Sparklite

### ISSUE THREAD

**Issue by EmulationFanatic** [Ryujinx/Ryujinx-Games-List#1914]
#### Current on `master` : 1.0.4767

Game loads and plays at a low to smooth FPS, depending on on-screen effects. Even when FPS is low, the game can be progressed through.

#### Screenshots :

![2020-07-01 16_34_21-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297972-e5fb0a80-bbb9-11ea-9285-d30d8dc1fd7e.png)
![2020-07-01 16_34_27-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297974-e72c3780-bbb9-11ea-9f8c-45d977ae6290.png)
![2020-07-01 16_34_57-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297978-e72c3780-bbb9-11ea-9382-512f566ab602.png)
![2020-07-01 16_36_55-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297981-e7c4ce00-bbb9-11ea-832d-240c02a8dfaa.png)
![2020-07-01 16_37_22-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297985-e8f5fb00-bbb9-11ea-9e59-f669a2168dc1.png)
![2020-07-01 16_38_44-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297989-e98e9180-bbb9-11ea-930e-847c04780921.png)
![2020-07-01 16_41_03-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297991-ea272800-bbb9-11ea-92ae-f341109dbf7f.png)
![2020-07-01 16_41_22-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297994-eb585500-bbb9-11ea-9490-30d02c2e00a6.png)
![2020-07-01 16_41_43-Ryujinx 1 0 4767 - Sparklite v1 0 0 (01007ED00C032000) (64-bit)](https://user-images.githubusercontent.com/62343878/86297995-ebf0eb80-bbb9-11ea-9394-5dba11c8b411.png)

#### Log file :

[Sparklite.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4861048/Sparklite.zip)

**Comment by F1ReB4LL** [comment]
## Sparklite

#### Game Update Version : *1.6.12*

#### Current on `master` : *1.1.208*

#### Graphics Backend : *Both Vulkan and OpenGL*

*Looks OK, haven't noticed any slowdowns.*

```
Last error returned: No errors shown
```

#### Hardware Specs : 

##### CPU: *i5-11400H*
##### GPU: *NVIDIA RTX 3050*
##### RAM: *24GB*

#### Screenshots :
![ryujinx_spl1](https://user-images.githubusercontent.com/39680140/183228152-3a97d29c-1bee-4fb1-8257-5a11df507258.png)
![ryujinx_spl2](https://user-images.githubusercontent.com/39680140/183228159-2aeebbcc-0558-436a-b5bb-da5cccfb0d22.png)
![ryujinx_spl3](https://user-images.githubusercontent.com/39680140/183228160-3a3a887d-57cf-4639-96cb-2903adf87011.png)
![ryujinx_spl4](https://user-images.githubusercontent.com/39680140/183228161-3e0d2fd0-abc5-440e-a3ff-eebc5c5b34bc.png)
![ryujinx_spl5](https://user-images.githubusercontent.com/39680140/183228163-93fc8ab1-f4da-4131-908a-29ebfdd2826b.png)
![ryujinx_spl6](https://user-images.githubusercontent.com/39680140/183228166-809b869f-f3b4-460f-8392-514fc9de06c3.png)
![ryujinx_spl7](https://user-images.githubusercontent.com/39680140/183228167-ae313d42-87dc-49fc-8e36-bb221491c2c6.png)

**Comment by MetrosexualGarbodor** [comment]
Is it meant to look that dark in the 2nd and 3rd screenshots?

**Comment by F1ReB4LL** [comment]
> Is it meant to look that dark in the 2nd and 3rd screenshots?

Yeah, it's due to a rainy storm that causes your airship to crash. There are occasional lightning flashes that make the screen brighter (4th screenshot).
